### PR TITLE
Just the change on sensitivity modifier for watch service

### PR DESCRIPTION
**Pull request by wibisono**
As you suggested, this PR is without refactoring, just to change the modifier to poll more frequently.

The change in .travis.yml is just to allow sbt to be access from testing/cache so I don't have to even modify build.sbt.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ script:
     echo "ensimeServerProjectJars := Nil" >> ensime.sbt ;
   - export JAVA_HOME=$(/usr/libexec/java_home)
   - curl https://raw.githubusercontent.com/paulp/sbt-extras/master/sbt -o sbt ; chmod 755 sbt
+  - export PATH=$PWD:$PATH
```
Comment (fommil) [inline .travis.yml]: not needed anymore

File: core/src/main/scala/org/ensime/filewatcher/FileWatchService.scala [hunk 2]
```diff
@@ -180,12 +182,8 @@ class FileWatchService { self =>
       log.trace(s"register ${dir} with WatchService")
     if (!observers.isEmpty) {
       val key: WatchKey = try {
-        dir.toPath.register(
-          watchService,
-          ENTRY_CREATE,
-          ENTRY_MODIFY,
-          ENTRY_DELETE
-        )
+        val events: Array[WatchEvent.Kind[_]] = Array(ENTRY_CREATE, ENTRY_MODIFY, ENTRY_DELETE)
```
Comment (fommil) [inline core/src/main/scala/org/ensime/filewatcher/FileWatchService.scala]: you need to run `sbt fmt` ... this is beyond the line limit.

**Comment by fommil** [comment]
if you do a reformat, this is good for merge. Thanks!

File: .travis.yml (hunk 1 above)
Comment (wibisono) [reply .travis.yml]: True, removed

File: core/src/main/scala/org/ensime/filewatcher/FileWatchService.scala (hunk 2 above)
Comment (wibisono) [reply core/src/main/scala/org/ensime/filewatcher/FileWatchService.scala]: sbt fmt'ed

**Comment by wibisono** [comment]
I did reformatted with sbt fmt, and remove the line on travis that is no longer needed.

**Comment by fommil** [comment]
unfortunately we no longer have any automatic publishing on merge as our drone server is out of action, sorry, but you can at least build your own version locally.

**Pull request merged**